### ISSUE TITLE
pathlib.Path.__enter__() is deprecated and scheduled for removal in Python 3.13

### ISSUE THREAD

**Issue by ldeluigi** [timvink/mkdocs-git-authors-plugin#68]
```
INFO     -  DeprecationWarning: pathlib.Path.__enter__() is deprecated and scheduled for removal in Python 3.13; Path objects as a context manager is a no-op
  File "/usr/local/lib/python3.11/site-packages/mkdocs_git_authors_plugin/ci.py", line 23, in raise_ci_warnings    
    with Path(path):
  File "/usr/local/lib/python3.11/pathlib.py", line 891, in __enter__
    warnings.warn("pathlib.Path.__enter__() is deprecated and scheduled "
```

**Comment by timvink** [comment]
Thanks for reporting, will pick this up!

**Comment by timvink** [comment]
fixed in 64cb4459c8c63b7192b74c0ca807142cb53e672f
